hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/application/ArticleService.kt

```diff
@@ -111,18 +154,47 @@ class ArticleService(
     }
 
     @Transactional(readOnly = true)
-    fun showAllByNickname(nickname: String): List<ArticleFeedResponse> {
-        val articles = userRepository.findByNicknameOrNull(nickname)
-            ?.let(articleRepository::findAllByAuthor)
+    fun showAllByNickname(nickname: String, page: Int, pageSize: Int): List<ArticleFeedResponse> {
+        val user: User = userRepository.findByNicknameOrNull(nickname)
             ?: EntityNotFoundException.notExistsNickname(User::class, nickname)
 
-        return articles
-            .filter { !it.anonymous }
+        val pageRequest: PageRequest = PageRequest.of(page, pageSize)
+        val articles = user
+            .let { articleRepository.findAllByAuthor(it, pageRequest) }
+
+        return articles.content.filter { !it.anonymous }
+            .map {
+                ArticleFeedResponse(
+                    article = it,
+                    articleOptions = articleOptionRepository.findAllByArticle(it),
+                    articleFavorites = articleFavoriteRepository.findAllByArticle(it),
+                    page = page,
+                    totalArticleCount = articleRepository.countAllByAuthor(user)
+                )
+            }
+    }
+
+    @Transactional(readOnly = true)
+    fun showAllByFavoritedUserId(userId: Long, page: Int, pageSize: Int): List<ArticleFeedResponse> {
```

Comment:
```suggestion
    fun showAllByFavoritedUserId(favoritedUserId: Long, page: Int, pageSize: Int): List<ArticleFeedResponse> {
```

변수명 이렇게 두면 좀 더 코드 읽을 때 편할 것 같어

Reply:
반영완료